jdaviz/core/freezable_state.py

```diff
@@ -53,6 +55,27 @@ def _reset_x_limits(self, *event):
             self.x_min = x_min
             self.x_max = x_max
 
+    def _convert_units_y_limits(self, old_unit, new_unit):
+        # override glue's _convert_units_y_limits to account
+        # for equivalencies.  This converts all four corners
+        # of the limits to set new limits that contain those
+        # same corners
+
+        if old_unit != new_unit:
+            if old_unit is None or new_unit is None:
+                self._reset_y_limits()
+                return
+
+            x_corners = np.array([self.x_min, self.x_min, self.x_max, self.x_max])
+            y_corners = np.array([self.y_min, self.y_max, self.y_min, self.y_max])
+            spec = Spectrum1D(spectral_axis=x_corners * u.Unit(self.x_display_unit),
+                              flux=y_corners * u.Unit(old_unit))
+            y_corners_new = flux_conversion(y_corners, old_unit, new_unit, spec)
+
+            #with delay_callback(self, 'y_min', 'y_max'):
```

Comment:
@astrofrog - any idea why including this `delay_callback` would result in `y_max` being immediately reverted to zero?

Reply:
weird, I don't see this as an issue anymore locally or in tests - I guess some other change may have affected it 🤷‍♂️   If anyone does see this happen, please let me know!